src/librustc_driver/lib.rs

```diff
@@ -852,6 +854,9 @@ impl RustcDefaultCalls {
                 PrintRequest::TargetCPUs | PrintRequest::TargetFeatures => {
                     rustc_trans::print(*req, sess);
                 }
+                PrintRequest::NativeStaticLibs => {
+                    println!("Native static libs can be printed only during linking");
```

Comment:
Could we use a warning mechanism here? Or anything other than a println, basically

Reply:
I've used `println` for consistency here, because all arms of this match already use `println`.

Reply:
I guess we should only hit these on `--print`